homeassistant/components/sensibo/climate.py

```diff
@@ -339,3 +361,10 @@ async def _async_set_ac_state_property(
         raise HomeAssistantError(
             f"Could not set state for device {self.name} due to reason {failure}"
         )
+
+    async def async_assume_state(self, state) -> None:
+        """Sync state with api."""
+        if state == self.state or (state == "on" and self.state != HVAC_MODE_OFF):
```

Comment:
I don't think we should have this check. We can't guarantee that we have the correct local state since we're a polling integration. So we should not stop the user from changing cloud state based on our local state.

Reply:
Removing this check is probably better, the user can make any condition in the automation itself which will probably be better and make the service have a fixed behavior.